om/test/DoubleTest.cpp

```diff
@@ -0,0 +1,55 @@
+#include <OMR/Infra/BitUtilities.hpp>
+#include <OMR/Infra/Double.hpp>
+
+#include <gtest/gtest.h>
+
+namespace OMR {
+namespace Infra {
+namespace Double {
```

Comment:
I used the Double namespace just to group the constants (maybe I should have used a class). I don't think it makes sense to use the double namespace here.